governance/votes/Executive vote - September 24, 2021.md

```diff
@@ -0,0 +1,73 @@
+  - **Increase** the WBTC-A [Local Liquidation Limit (ilk.hole)](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) from 15 million DAI to **25 million DAI**
+- Multiple **test actions** will take place for Layer 2 DAI solutions on Optimism and Arbitrum, with the intention of denying the Protocol Engineering Core Unit's Deployer Keys if successful. 
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Adjusting Auction Parameters for ETH-A, ETH-B, ETH-C, and WBTC-A
+
+As per the result of [this poll](https://vote.makerdao.com/polling/QmfGk3Dm?network=mainnet#poll-detail), the following changes to Auction Parameters for ETH-A, ETH-B, ETH-C, and WBTC-A will occur if this executive proposal passes.
+
+- **Decrease** the [Auction Price Multiplier (buf)](https://makerdao.world/en/learn/governance/param-auction-price-multiplier) for ETH-A, ETH-C, and WBTC-A vaults from 1.3 to **1.2**
+- **Increase** the [Local Liquidation Limit (ilk.hole)](https://makerdao.world/en/learn/governance/param-local-liquidation-limit) as detailed below:
+  - ETH-A Local Liquidation Limit from 30 million DAI to **40 million DAI**
+  - ETH-B Local Liquidation Limit from 15 million DAI to **25 million DAI**
+  - ETH-C Local Liquidation Limit from 20 million DAI to **30 million DAI**
+  - WBTC-A Local Liquidation Limit from 15 million DAI to **25 million DAI**
+
+For more information on the rationale behind these proposed changes, please view [this post](https://forum.makerdao.com/t/adjusting-auction-parameters-for-eth-a-eth-c-and-wbtc-a-vault-types/10331) from the Risk Core Unit. 
+
+### Layer 2 Test Actions
+
+If this executive proposal passes, it will test the configuration of L2 components, including the governance relay and L2 DAI. Once it is confirmed that Governance has control over all L2 systems, the Protocol Engineering Core Unit will **manually** `deny` its deployer addresses, meaning Governance will be the only authority retaining full control of the token bridges.
```

Comment:
Should we have a link to a PE forum post for this?

Reply:
Like even I'm not 100% sure what we're doing from this explanation. I think:
* Ensuring that Maker Governance is able to make changes to DAI bridges on both sides.

What are we testing about L2 DAI? I wasn't aware we could do anything to it?

Reply:
I think this covers it, post is up and now linked